docs/guide-send-tx-bundle.mdx

```diff
@@ -0,0 +1,33 @@
+---
+title: Guide to Sending Transactions and Bundles on Flashbots
+---
+
+Navigating the various methods for submitting transactions and bundles on Flashbots can be overwhelming. This guide aims to clarify the available options, helping you choose the most suitable one.
+
+## Understanding Relay and RPC Endpoints
+
+Flashbots offers two primary JSON-RPC endpoints on the Ethereum Mainnet: `rpc.flashbots.net` and `relay.flashbots.net`.
+
+### `rpc.flashbots.net` - For Retail Users
+- **Purpose**: Designed specifically for retail users as drop-in RPC replacement for MEV protection and refund.
```

Comment:
Nit: Would simplify the purpose to just -- "Designed specifically for retail users as a drop-in RPC replacement in their wallet."

We already mention protection and refunds in the features section so this seems duplicative.